build/build.ps1

```diff
@@ -82,15 +82,13 @@ if (Get-Command nuget -ErrorAction SilentlyContinue) {
                     throw
                 }
             } Catch {
-                Write-Host "##vso[task.logissue type=error;]Failed to build VS extension."
-                $all_ok = $False
+                Write-Host "##vso[task.logissue type=warning;]Failed to build VS extension."
```

Comment:
The issue is that nuget will use the msbuild version that VS uses, and that one will error because it's a 2.* version that does not support targeting 3.0. Ideally we get a build setup that used VS 16.3 and then all works as is.